cilium-ubuntu.json

```diff
@@ -1,13 +1,13 @@
 {
   "variables": {
     "box_basename": "ubuntu-17.10",
-    "cpus": "2",
+    "cpus": "8",
```

Comment:
IMO, this is far too high. Why the change from 2 to 8? How this affect developers - is this just the default? I.e., will changing the configuration fo how this VM is booted in the Vagrantfile take precedence over this value?

Reply:
It's in the commit message
```

Those values can be a higher number to have a faster build. Later on,
each Vagrantfile can decrease the number of CPUs and memory to use.

Signed-off-by: André Martins <andre@cilium.io>
```

Reply:
My bad, looked at the file changes and not the commits.